src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,68 @@
+package duke.task;
+
+import java.io.IOException;
+
+public abstract class Task {
+
+    /**
+     * The description of the Task
+     */
+    protected String descr; //descr as an abbreviation for description
```

Comment:
Consider standardising "descr" to its full form for consistency. e.g the Todo class uses "description".